lib/Common/Memory/Recycler.h

```diff
@@ -1348,6 +1348,10 @@ class Recycler
     template <ObjectInfoBits infoBits>
     char * AllocVisitedHost(DECLSPEC_GUARD_OVERFLOW size_t size)
     {
+        // We need to track these allocations. The RecyclerVisitedHost* object allocation APIs don't provide us with the type of the objects being allocated.
+        // Use a dummy type to track the allocations.
```

Comment:
None of the Recycler::Alloc* function implicit do TRACK_ALLOC_INFO. 

You should follow the pattern of the others and modify the RecyclerAllocVisited* marcro to do TRACK_ALLOC_INFO as well.